### ISSUE TITLE
perl 5.24 compatibility

### ISSUE THREAD

**Issue by remicollet** [squentin/gmusicbrowser#135]
Current version 1.1.5 is not compatible with latest perl.

```
Can't redeclare "my" in "my" at /usr/bin/../share/gmusicbrowser/gmusicbrowser_list.pm line 8001, at end of line
Compilation failed in require at /usr/bin/gmusicbrowser line 440.
```

and 

```
Can't redeclare "my" in "my" at /usr/bin/gmusicbrowser line 8980, at end of line
BEGIN not safe after errors--compilation aborted at /usr/bin/gmusicbrowser line 9091.
```

I notice both are fixed here.

Would it be possible to plan a new release ?
(I can apply the patches, but this have some risk I forget some part)

**Comment by remicollet** [comment]
For now 853840eb9dad0b59ad2dac5d303f5929b2f09f21 seems enough.

**Comment by bor** [comment]
Here is link to current 'project status' issue
https://github.com/squentin/gmusicbrowser/issues/149

**Comment by dmhursh** [comment]
This appears to be fixed in the `v1.1.x` branch.  Can it be closed?
